.gitignore

```diff
@@ -10,4 +10,7 @@ venv
 .env
 bigtable-admin.json
 bigtable-writer.json
+DEPLOY-*
+VAA-VERIFY-*
```

Comment:
Do these have to be at the top level, or can we move them to staging/algorand/.gitignore?

Reply:
We can even remove those items, as they are deployment-only output files.